README.md

```diff
@@ -1,10 +1,155 @@
 # Community
 
-## Become a member of rust-vmm
+## What is rust-vmm
+
+rust-vmm is an open-source project that enables the community to build custom
+Virtual Machine Monitors (VMMs) and Hypervisors. It offers a base of
+virtualization packages that other projects can use for rapidly developing
+virtualization solutions while focusing on the key differentiators of their
+product rather than re-implementing common components like KVM wrappers, Virtio
+devices and other VMM libraries.
+
+The rust-vmm project is organized as a shared-effort, shared-ownership
+open-source project that includes (so far) contributors from Alibaba, AWS,
+Cloud Base, Crowdstrike, Intel, Google, Red Hat as well as individual
+contributors.
+
+Each virtualization component lies in a separated GitHub repository under the
```

Comment:
s/component/package/ (for consistency with above)
s/lies/lives/

Reply:
modified the above mention with component.